test/parallel/test-fs-realpath-buffer-encoding.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const fs = require('fs');
 
-const string_dir = fs.realpathSync(common.fixturesDir);
+const string_dir = fs.realpathSync(fixtures);
```

Comment:
Fixtures module is exporting object, this change is failing tests.

Reply:
this should be `const string_dir = fs.realpathSync(fixtures.fixturesDir);`